packages/api-sdk/src/resources/projectSecret.ts

```diff
@@ -0,0 +1,38 @@
+import type Fetch from '@api-sdk/fetch';
+import { BaseModels } from '@voiceflow/base-types';
+
+import BaseResource from './base';
+
+const ENDPOINT = 'projects';
+
+class ProjectSecretResource extends BaseResource<ProjectSecretResource> {
+  constructor(fetch: Fetch) {
+    super({ fetch, clazz: ProjectSecretResource, endpoint: ENDPOINT });
+  }
+
+  public async read(projectID: string, tag: number): Promise<string> {
+    const { data } = await this.fetch.get<string>(`${this.endpoint}/${projectID}/secret/${tag}`);
+
+    return data;
+  }
+
+  public async find(lookup: string, tag: number): Promise<BaseModels.ProjectSecret.Model[]> {
+    const { data } = await this.fetch.get<BaseModels.ProjectSecret.Model[]>(
+      `${this.endpoint}/secret?lookup=${encodeURIComponent(lookup)}&tag=${tag}`
+    );
+
+    return data;
+  }
```

Comment:
Just wondering, do you think it's possible to better differentiate these two functions using their names?
Maybe `findByProjectID` vs `findByLookup` or something.

Reply:
sure, will use that suggestion